### PR TITLE
Add notes on deleting an operator CR

### PR DESCRIPTION
**Pull request by awgreene**
This commit introduces documentation detailing how to delete an operator
CR.

Signed-off-by: Alexander Greene <greene.al1991@gmail.com>

### CODE REVIEW

File: content/en/docs/Tasks/uninstall-operator.md [hunk 1]
```diff
@@ -67,3 +67,13 @@ kubectl delete csv $CSV -n <namespace>
...
+
+## Step 5: Deleting the Operator CR
+
+OLM recently introduced the view-only [operator CRD](https://github.com/operator-framework/api/blob/7339a22050af53df7b6f97a652b8e2d73698765a/crds/operators.coreos.com_operators.yaml). For those of you unfamiliar with the resource, when OLM installs an operator:
+- All resources associated with the operator will have the `operators.coreos.com/<operator CR Name>` label applied to it.
+- OLM will create or recreate an `operator CR` if any resources exists with the `operators.coreos.com/<operator CR Name>` label.
+
+This ultimately means that in order to delete the `operator CR` users will need to delete all entries in its `status.Components` array. Typically, an `operator CR` can be delete after completing steps 1 through 4 above. However, if a user is still unable to delete an operator, they should:
```
[the remark this answers is not in the document]
Comment (awgreene) [reply content/en/docs/Tasks/uninstall-operator.md:77]: Done!

File: content/en/docs/Tasks/uninstall-operator.md [hunk 2]
```diff
@@ -67,3 +67,13 @@ kubectl delete csv $CSV -n <namespace>
...
+1. Delete each resource found in the `operator CR's` status.Components array. Alternatively, if you have deleted the operator's CSV and Subscription, you can simply remove the `operators.coreos.com/<operator CR Name>` label.
+2. Delete the `operator CR`.
```
Comment (timflannagan) [inline content/en/docs/Tasks/uninstall-operator.md:79]: These bullet points don't seem to render in the deploy preview. It looks like you'd need to substitute them with the `-` character.

**Review by perdasilva — APPROVED** [review]

**Pull request merged**